recipes/omero-rdf/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "omero-rdf" %}
+{% set version = "0.6.0" %}
+{% set python_min = "3.9" %}
```

Comment:
```suggestion
```

Reply:
I don't understand what to do here?
Why would I remove this line? The docs recommend to have a `python_min`:
>     * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
>       
>       * For the `host` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.
>       * For the `run` section of the recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
>       * For the `test.requires` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.

Reply:
3.9 is the default. You only add this if your min is higher than conda-forge's min, say >=3.10